docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx

```diff
@@ -0,0 +1,157 @@
+
+---
+
+## PagerDuty + Dagster Cloud integration benefits
+
+- Empower your team with real-time incident response by integrating Dagster Cloud and PagerDuty
+- Notify on-call responders when alerts are triggered in Dagster Cloud
+- Dramatically reduce downtime for issues related to asset materialization, asset checks, job runs, schedules, sensors, and code locations
+- Customize the urgency and routing of PagerDuty incidents based on the content of the Dagster Cloud event payload
+
+---
+
+## How it works
+
+- Events that trigger an alert in Dagster Cloud will automatically create a PagerDuty incident
+- Incidents can be triggered for events related to runs, assets, schedules, sensors, and code locations
+- Users must resolve incidents in PagerDuty. Dagster Cloud doesn't currently automatically resolve incidents.
+
+### Requirements
+
+To integrate PagerDuty with Dagster Cloud, you'll need:
+
+- **Permissions in PagerDuty that allow you to manage services and integrations.** Refer to PagerDuty's [User roles](https://support.pagerduty.com/docs/user-roles) documentation for more information.
+- **The following in Dagster Cloud**:
+  - **A Pro plan**
+  - **Organization, Admin, or Editor permissions**, which are required to create and manage alerts
+
+### Support
+
+If you need help with this integration, contact Dagster Cloud support.
+
+---
+
+## Integration walkthrough
+
+### In PagerDuty
+
+1. From the **Configuration** menu, click **Services > Service Directory**.
+
+2. There are two ways to add an integration to a service:
+
+   - **If adding your integration to an existing service**: Click the **name** of the service you want to add the integration to. Then, select the **Integrations** tab and click the **New Integration** button.
+
+   - **If creating a new service for your integration**: Refer to PagerDuty's [Configuring services and integrations documentation](https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations) and follow the steps outlined in the [Create a new service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting **Dagster Cloud** as the **Integration Type** in step 4. Continue with the [**In Dagster Cloud**](#in-dagster-cloud) section (below) once finished.
+
+3. Enter an **Integration Name** in the format `monitoring-tool-service-name` (e.g. `dagster-cloud-run-failures`) and select **Dagster Cloud** from the **Integration Type** menu.
+
+4. Click the **Add Integration** button to save your new integration. You will be redirected to the **Integrations** tab for your service.
+
+5. An **Integration Key** will be generated on this screen. Keep this key saved in a safe place, as it will be used when you configure the integration with **Dagster Cloud** in the next section.
+
+   ![](https://pdpartner.s3.amazonaws.com/ig-template-copy-integration-key.png)
```

Comment:
I'm going to replace this - just wanted to get the draft in front of you more quickly.